edn_format/immutable_list.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import collections
+
+
+class ImmutableList(collections.MutableSequence, collections.Hashable):
+    def __init__(self, wrapped_list, copy=True):
+        self._list = list(wrapped_list) if copy else wrapped_list
+        self._hash = None
+
+    def __repr__(self):
+        return self._list.__repr__()
+
+    def __eq__(self, other):
+        if isinstance(other, ImmutableList):
+            return self._list == other._list
+        else:
+            return self._list == other
+
+    def _call_wrapped_list_method(self, method, *args):
+        new_list = list(self._list)
```

Comment:
Minor nit: Thoughts on using `copy.copy` instead of calling `list()`? Explicit is better than implicit?